Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/187801
+@@||res.ads.nicovideo.jp^|
```

Comment:
What about to replace 
```
||ads.nicovideo.jp^
```
by
```
://ads.nicovideo.jp^
```
+ for DNS (one `|`)
```
|ads.nicovideo.jp^
```
?

Reply:
I'm not sure whether the only subdomain is `res`, and tend to think the exception will eventually become unnecessary. It's safer to keep `||ads.nicovideo.jp^` and only allow what's causing a problem.

Reply:
but we need to unblock only `ads.nicovideo.jp`, without subdomains of `ads.`

Reply:
![image](https://github.com/user-attachments/assets/735ef839-7387-4fa3-a6a3-4593827fa7b5)
Another their subdomains - only one mentioned in our filters.